com.playeveryware.eos/Documentation~/full_guide/01_starthere.md

```diff
@@ -0,0 +1,126 @@
+<a href="/README.md"><img src="/com.playeveryware.eos/Documentation~/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
+
+# Full Guide to Implementing the EOS Plugin for Unity
+
+This document serves as the starting point for a beginning-to-end guide on how to implement the EOS Plugin for Unity into your game.
+The only prerequisite to following this guide is having a game you want to implement the plugin into. This will walk you through
+the steps to set up your project, with links to documentation for more details and information.
+
+# Signing Up for Epic Online Services
+
+The EOS Plugin for Unity requires you to have a configured account and game on the Epic Game Store portal.
+For information about the benefits of publishing through Epic, see [Epic's Store Distribution portal](https://store.epicgames.com/en-US/distribution).
+This leads to [the Developer Portal landing page](https://dev.epicgames.com/portal/en-US/).
+Pressing Sign In will present you with a screen that you can sign up for a new account.
+If you already have an Epic Games Account, perhaps because you've played games on the Epic Games Store, you can sign in with that existing account.
+
+<a href="https://www.epicgames.com/id/login?"><img src="/com.playeveryware.eos/Documentation~/full_guide/epic_developerportal_signin_or_createaccount.png" alt="You can 'create account' on the Epic Developer Portal."/></a>
+
+Upon signing up or creating an account, you'll be required to create an Organization.
+An Organization is a holder for all of the Products you make. See [Epic's documentation on Organizations](https://dev.epicgames.com/docs/dev-portal/organization-management) for more information.
+
+## Working With an Existing Organization
+
+If you or a team member has already created an Organization, you can add Epic Game Store accounts to that organization.
+While signed in to the `Epic Games Developer Portal`, navigate to `Organization`.
+Under the `Members` tab, you can use the `Invite New` button to send an invitation email to the target user.
+That email will contain a link that will help the user sign up for an Epic Account if necessary, and then they'll join your organization.
+
+Everyone who will be testing your game should be a part of your organization, or part of a Player Group. If they aren't, then whenever the EOS SDK requires the user to login, that user will be forbidden from accessing your application.
```

Comment:
I disagree in that the documentation on Player Groups is shared _immediately_ following this sentence.

This also explains why it is necessary, because it says that if a user isn't part of a Player Group then they can't use the application. I'm not sure what to change to make this more clear.